src/cli_shared/cli/mod.rs

```diff
@@ -93,11 +92,11 @@ pub struct CliOpts {
     /// pre-loaded database
     #[arg(long)]
     pub skip_load: Option<bool>,
-    /// Number of tipsets requested over chain exchange (default is 200)
+    /// Number of tipsets requested over one chain exchange (default is 8)
     #[arg(long)]
-    pub req_window: Option<i64>,
+    pub req_window: Option<usize>,
```

Comment:
I believe this option was not working because `config.sync.req_window` was never used.